Form/Type/ColorSelectorType.php

```diff
@@ -16,6 +16,17 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ColorSelectorType class is deprecated since version 3.4 and will be removed in 4.0.'
```

Comment:
Next time just leave 3.x, **do not fill the version**.

Reply:
@Soullivaneuh thank you for commenting and explaining what was wrong on my side, will have that stuff in mind if I make a PR in future.